src/App.js

```diff
@@ -21,18 +21,19 @@ function App() {
     setCartCount((prev) => prev + 1);
   }
 
-  function handleCartOpen() {
+  function handleToggleCart() {
```

Comment:
Hi @sandrina-p, so here are the two bugs I haven't been able to solve: 

First, this doesn't work to close the cart. The first click opens it, the second causes it to briefly close and then reopen. I don't get what's causing it to re-render twice

Reply:
There are two event listeners. One is when clicking the menu link, the other is clicking outside the cart.

So when you click to close, you are triggering both events. (if you add a console log to both functions, you'll notice that both are called).

Reply:
Of course! Thanks! I'm using react-onclickoutside, it has the functionality to ignore certain classes but I had to open an issue to see how to implement it, not much about it in the docs.

Reply:
My quick around suggestion is to  ignore the click it it's open.